src/Mutts.jl

```diff
@@ -73,34 +73,44 @@ actions right before an object is branched.
 branchactions(obj) = nothing
 
 """
-    markimmutable!(obj)
+    mark_immutable!(obj)
 
 Freeeze `obj` from further mutations, making it eligible to pass to
 other Tasks, branch! from it, or otherwise share it.
 """
-function markimmutable! end
+function mark_immutable! end
 
-markimmutable!(o::T) where T = markimmutable!(mutts_trait(T), o)
+mark_immutable!(o::T) where T = mark_immutable!(mutts_trait(T), o)
 
-markimmutable!(::NonMuttsType, a) = a
+mark_immutable!(::NonMuttsType, a) = a
 
-@generated function markimmutable!(::MuttsType, obj :: T) where T
+@generated function mark_immutable!(::MuttsType, obj :: T) where T
```

Comment:
```suggestion
@generated function mark_immutable!(::MuttsType, obj::T) where T
```
Also, these spaces were old style